samples/drivers/dac/src/main.c

```diff
@@ -21,6 +21,10 @@
 #define DAC_DEVICE_NAME		DT_LABEL(DT_NODELABEL(dac0))
 #define DAC_CHANNEL_ID		0
 #define DAC_RESOLUTION		12
+#elif defined(CONFIG_BOARD_ARDUINO_ZERO)
+#define DAC_DEVICE_NAME		DT_LABEL(DT_NODELABEL(dac0))
+#define DAC_CHANNEL_ID		0
+#define DAC_RESOLUTION		10
```

Comment:
Any idea the why this two properties are not on dac-controller.yaml?
This could remove this if-def thing.

Reply:
Nope, I don't know.

FWIW, Linux doesn't either and instead has an ioctl `IIO_CHAN_INFO_SCALE` for reading the vref and scale at runtime.

Reply:
I think is reasonable add channel id and resolution to dac-controller.yaml.
What do you think?

Reply:
What shall we do next with this?  WDYT about splitting the resolution issue out and addressing it for all DACs?  It may need API review as if we're putting the resolution in the DAC io-channel then we should discuss doing the same in the ADC io-channel.

Reply:
We can discuss it in the new issue #27191 and afterwards get it into the API meeting for a decision.